src/OpenCensus.Collector.AspNetCore/RequestsCollector.cs

```diff
@@ -41,7 +41,9 @@ public RequestsCollector(RequestsCollectorOptions options, ITracer tracer, ISamp
         {
             this.diagnosticSourceSubscriber = new DiagnosticSourceSubscriber(
                 new Dictionary<string, Func<ITracer, ISampler, ListenerHandler>>()
-                { { "Microsoft.AspNetCore", (t, s) => new HttpInListener(t, s, propagationComponent) } },
+                {
+                    { "Microsoft.AspNetCore", (t, s) => new HttpInListener(t, s, propagationComponent) },
```

Comment:
can you reference a type as opposed to hardcoded string?

Reply:
No. One of ideas of DiagnosticsSource is ducktyping and eliminating direct referencens between packages. I added comment where I use reflection